backend/app/management/commands/runanalysis.py

```diff
@@ -68,7 +68,8 @@ def handle(self, *args, **options):
                 model_instances = model.objects.all()
 
             for model_instance in model_instances:
-                print(f'Running {analysis_name} on {model} {model_instance.id}')
+                print(f'Running {analysis_name} on {model} {model_instance.id} '
+                      f'({model_instance.number}_{model_instance.map_square.number})')
```

Comment:
Can you make the text printed here more verbose (e.g, `Map square: X, Photo Number: Y`)? That'll help, too.

Reply:
@ryaanahmed Hi Ryaan, I made the requested changes.

Reply:
Thanks -- looks good.